contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -255,6 +256,10 @@ contract OETHVaultCore is VaultCore {
 
         require(request.claimed == false, "already claimed");
         require(request.withdrawer == msg.sender, "not requester");
+        require(
+            request.timestamp + CLAIM_DELAY <= block.timestamp,
+            "claim delay not met"
```

Comment:
nit. start with a capital letter in error messages

Reply:
We decided in the contracts meeting a month or so ago that we would not introduce custom errors to the dollar repo. Better to not mix the approaches. We can use custom errors on the new repos/products